assets/tests/hooks/useVehicles.test.ts

```diff
@@ -88,10 +88,7 @@ describe("useVehicles", () => {
 
     const { result } = renderHook(() => useVehicles(mockSocket, ["1"]))
 
-    expect(result.current).toEqual({
-      "1": [{ ...vehicle, timestamp: null }],
-    })
-
+    expect(result.current).toEqual({})
```

Comment:
question: I dunno if it's worth asserting that it's equal to a empty object, but that is the default value when there is no data.